PubSub/tests/Unit/TopicTest.php

```diff
@@ -225,28 +225,29 @@ public function testPublishBatch()
         $this->assertEquals($res, $ids);
     }
 
-    public function testPublishBatchUnencoded()
-    {
-        $message = [
-            'data' => 'hello world',
-            'attributes' => [
-                'key' => 'value'
-            ]
-        ];
-
-        $this->connection->publishMessage(Argument::allOf(
-            Argument::withEntry('foo', 'bar'),
-            Argument::withEntry('messages', [$message]),
-            Argument::that(function ($options) use ($message) {
-                // If the message was encoded, this will fail the test.
-                return $options['messages'][0]['data'] === $message['data'];
-            })
-        ));
-
-        $this->topic->___setProperty('connection', $this->connection->reveal());
-
-        $res = $this->topic->publishBatch([$message], ['foo' => 'bar', 'encode' => false]);
-    }
+    // // *** THIS TEST HAS NEVER WORKED AND THE FUNCTIONALITY DOESNT EXIST ***
```

Comment:
Callout for this test - when I looked, there was no support for a "encode" option to be set to false in the `publishBatch` method that I could find. So I commented out the test. 

cc @saranshdhingra @dwsupplee

Reply:
The encode setting flows down from the top level client: https://github.com/googleapis/google-cloud-php/blob/main/PubSub/src/PubSubClient.php#L160-L166. Users shouldn't need to worry about using it. As for the note about the test never working, I'll need to look a bit more into that.

Reply:
This test tries to pass an encode option at method call time (`publishBatch` with `'encode' => false`). I recommend removing the test altogether, as the functionality doesn't exist, and since users don't need to worry about using it, it's not functionality we want to add.